api/ccapi/include/optimizer.h

```diff
@@ -35,6 +35,7 @@ class LearningRateScheduler;
 enum OptimizerType {
   ADAM = ML_TRAIN_OPTIMIZER_TYPE_ADAM,      /** adam */
   SGD = ML_TRAIN_OPTIMIZER_TYPE_SGD,        /** sgd */
+  ADAMW = ML_TRAIN_OPTIMIZER_TYPE_ADAMW,    /** AdamW */
   UNKNOWN = ML_TRAIN_OPTIMIZER_TYPE_UNKNOWN /** unknown */
 };
```

Comment:
Wouldn't it be better to write in alphabetical order unless there's a specific reason?

```suggestion
enum OptimizerType {
  ADAM = ML_TRAIN_OPTIMIZER_TYPE_ADAM,      /** adam */
  ADAMW = ML_TRAIN_OPTIMIZER_TYPE_ADAMW,    /** AdamW */
  SGD = ML_TRAIN_OPTIMIZER_TYPE_SGD,        /** sgd */
  UNKNOWN = ML_TRAIN_OPTIMIZER_TYPE_UNKNOWN /** unknown */
};
```

Reply:
Good idea. It wasn't particularly clear that that was the case, with only two entries.